.travis.yml

```diff
@@ -10,21 +12,28 @@ matrix:
     dist: trusty
     env: BUILDTYPE=Release DIST=trusty
   - os: osx
-    osx_image: xcode6.4
+    osx_image: xcode7.3
     env: BUILDTYPE=Debug
+    cache:
+      ccache: true
+      timeout: 360 # 6min timeout for cache uploading instead default 3
+      directories:
+        - $HOME/Library/Caches/Homebrew   # cache Homebrew ressources
   - os: osx
-    osx_image: xcode6.4
+    osx_image: xcode7.3
```

Comment:
If you're running 10.10 I'm pretty sure you have security exploits present, so I'm fine with this